sourcegraph/manifest.json

```diff
@@ -0,0 +1,57 @@
+{
+  "integration_id": "sourcegraph-services-map",
+  "categories": [
+    "monitoring",
+    "collaboration",
+    "source control"
+  ],
+  "app_uuid": "19916057-6b27-4c17-89c5-821794532315",
+  "creates_events": false,
+  "display_name": "Sourcegraph Datadog Services Map Extension",
+  "manifest_version": "2.0.0",
+  "guid": "b647480e-bdfb-4cf6-baa2-e773f69ac466",
+  "is_public": true,
+  "maintainer": "support@sourcegraph.com",
+  "name": "sourcegraph",
+  "public_title": "Sourcegraph Datadog Services Map Extension",
+  "short_description": "View services dependencies as tooltips on your source code",
+  "description": "View services dependencies as tooltips on your source code on Sourcegraph and your code host",
+  "support": "contrib",
+  "supported_os":  [
+    "linux",
+    "mac_os",
+    "windows"
+  ],
+  "classifier_tags":[], 
+  "author": {
+    "homepage": "https://www.sourcegraph.com",
+    "name": "Sourcegraph",
+    "sales_email": "sales@sourcegraph.com",
+    "support_email": "support@sourcegraph.com"
+  },
+  "display_on_public_website": true,
+  "app_id": "sourcegraph",
+  "type": "check",
+  "assets": {
+  },
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "View services dependencies as tooltips on your source code",
+    "title": "Sourcegraph Datadog Services Map Extension",
+    "media":[
+      {
+        "media_type": "image",
+        "caption": "Example of tooltip on Sourcegraph",
+        "image_url": "images/sourcegraph-datadog-services-map-extension.png"
+      }
+    ],
+    "classifier_tags": [
```

Comment:
I wasn't exactly sure what classifier tags to use here, but manifest validations required having them. I put the supported OS's after looking at what Sourcegraph is. Feel free to update this if this is wrong/more should be added!